test/NuGet.Core.Tests/NuGet.CommandLine.XPlat.Test/BasicLoggingTests.cs

```diff
@@ -69,6 +69,29 @@ public void BasicLogging_RestoreHelp_ExitCode()
         }
 
         [Fact]
+        public void BasicLogging_RestoreConfigFile_ExitCode()
+        {
+            // Arrange
+            var log = new TestLogger();
```

Comment:
In my mind, this test does more than just testing the negative.  It:

1. Ensures the "--configfile" doesn't fail because it is an unrecognized option.
2. NuGet is at least attempting to look for the file passed in with --configfile.